source/_integrations/autoskope.markdown

```diff
@@ -0,0 +1,179 @@
+---
+title: "Autoskope"
+description: "Instructions on how to integrate Autoskope devices into Home Assistant."
+ha_category:
+  - Device tracker
+ha_release: 2025.5
+ha_iot_class: cloud_polling
+ha_domain: autoskope
+ha_platforms:
+  - device_tracker
+ha_config_flow: true
+ha_integration_type: integration
+ha_codeowners:
+  - '@mcisk'
+ha_quality_scale: platinum
+---
+
+The **Autoskope** {% term integration %} allows you to integrate your [Autoskope](https://autoskope.de) vehicle tracking devices with Home Assistant. Autoskope provides GPS tracking devices and cloud services for vehicles and other assets, offering features like location tracking, voltage monitoring, and various alerts.
+
+This integration connects to the Autoskope cloud service to retrieve the current status and location of your registered vehicles.
+
+## Supported devices
+
+This integration should work with all past, present, and future hardware devices offered by Autoskope.
+
+## Prerequisites
+
+Configuration is handled entirely through the UI flow during setup. You will need to provide:
+
+* **Username:** Your Autoskope account username.
+* **Password:** Your Autoskope account password.
+* **Host:** URL of the Autoskope API server (e.g., `https://portal.autoskope.de`).
```

Comment:
```suggestion
- **Username**: Your Autoskope account username.
- **Password**: Your Autoskope account password.
- **Host**: URL of the Autoskope API server (for example, `https://portal.autoskope.de`).
```
- we use - instead of * for lists
- [avoid e.g.](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg) and prefer for example or such as instead